src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/ValidationContext.cs

```diff
@@ -28,12 +29,12 @@ public class ValidationContext
         /// <summary>
         /// The <see cref="CatalogIndexEntry"/>s for the package that were collected.
         /// </summary>
-        public IEnumerable<CatalogIndexEntry> Entries { get; }
+        public List<CatalogIndexEntry> Entries { get; }
```

Comment:
How about `IReadOnlyList<T>` instead here and below?

Reply:
Sounds good, fixed